src/_js/main.mjs

```diff
@@ -11,6 +11,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import * as DarkModeToggle from '/_js/dark-mode-toggle.min.mjs';
```

Comment:
Can this be just `import '/_js/dark-mode-toggle.min.mjs';`?

Reply:
Ooops, sure. https://github.com/v8/v8.dev/pull/254/commits/2c89589b24279015498b7706163800c40bd7098b